### DISCUSSION TITLE
Min Temp For Extruder

### DISCUSSION THREAD

**Discussion by robindonne1** [mriscoc/Ender3V2S1#192] [General]
Ok just had a bed evening/night figuring out why there was no extrusion during my prints.

I was busy calibrating a new extruder mounting plate and had to change my filament during this proces. change from PLA to Petg to check the cooling fans position.
While unloading the filament i realised the Heater was heating to 180 celsius while i had to unload PETG. So i scrolled through the menu to see if there was option to Select different Filament to load and unload. I found the "MinimumExtruderTemp"setting and changed it to 200 Celsius assuming this was more safe to move PETG in and out the hotend.

I unloaded the PETG, loaded the PLA and wanted to start a test print.
 
I openen a calibration cube in Cura and checked some setting.
i did remove some extra options from the Cura profiles, not checked or unchecked them, just remove the option to be choosen because most of them were never used by me.

Sliced the cube and send to octoprint.
No extrusion?
Checked the extruder again to see it would load and unload, all worked fine.

A lot of hours of slicing in multiple profiles with no result. Changed to PETG and the PETG profiles and all worked ok.
I assumed i had checked some boxes while i was removing hidden profile options.

Removed Cura complete, removed all the profiles, almost removed and smashed my Computer through the window.

I just realized that the "MinExtruderTemp" might have blocked all the extrusion from prints that are sliced with a lower Hotend temp compared the temp set in the Firmware.

Was i just stupid and is this option safe to have the extruder load all filament types on one temperature setting?
Or am i missing a setting where you can select which filament you want to unload?

i cant remember having the same issues with the Jyers firmware.

**Comment by mriscoc** [comment]
For Marlin, MinExtruderTemp is the minimum temperature necessary for allow extruder movements. When you go to filament management and request a load/unload/change filament, if the printer detects that you did not preheat the hot-end, it starts to heating to that temperature to allow extruder movements.

**Reply by robindonne1** [reply]
Yes, and that works perfect.  
I have set it to 180 now, but previously changed it to 200 because of the PETG filament load and unload.

What surprised me was that my objects, sliced in a PLA Cura profile with a hotend temp of 185, started to print but without extruding. 
All was related to my minimum temp for the extruder.

It would have been great if i could choose to load/unload 3 different filaments.

**Reply by mriscoc** [reply]
Noted for future releases
